sn_client/src/api/mod.rs

```diff
@@ -192,6 +195,15 @@ impl Client {
             .has_key(section_key)
     }
 
+    /// NetworkPrefixMap used to bootstrap the client on the network.
+    ///
+    /// This is updated by the client as it receives Anti-Entropy/update messages from the network.
+    /// Any user of this API is responsible for caching it so it can use it for any new `Client`
+    /// instance not needing to obtain all this information from the network all over again.
+    pub async fn get_prefix_map(&self) -> NetworkPrefixMap {
```

Comment:
Rust convention (https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter):

```suggestion
    pub async fn prefix_map(&self) -> NetworkPrefixMap {
```